dspace-api/src/main/java/org/dspace/app/util/SubmissionConfig.java

```diff
@@ -72,22 +66,13 @@ public SubmissionConfig(String submissionName, List<Map<String, String>> steps,
             Map<String, String> stepInfo = steps.get(stepNum);
             SubmissionStepConfig step = new SubmissionStepConfig(stepInfo);
 
-            // Only add this step to the process if either:
-            // (a) this is not a workflow process OR
-            // (b) this is a workflow process, and this step is editable in a
-            // workflow
-            if ((!this.isWorkflow)
-                    || ((this.isWorkflow) && step.isWorkflowEditable()))
```

Comment:
Just to be sure: We remove the workflow check here in the backend because this check should happen in the front-end instead?

Reply:
No, please take in mind that this PR it's only the first stage, to move forward quickly. 
A new issue will be the target to works togheter and define how to expone workflow submission for validators.

Reply:
Ok, thanks for the clarification.